twinkle.js

```diff
@@ -281,7 +281,16 @@ Twinkle.addPortlet = function(navigation, id, text, type, nextnodeid) {
 			if (navigation !== 'portal' && navigation !== 'left-navigation' && navigation !== 'right-navigation') {
 				navigation = 'mw-panel';
 			}
-			outerNavClass = 'mw-portlet vector-menu vector-menu-' + (navigation === 'mw-panel' ? 'portal' : type === 'menu' ? 'dropdown vector-menu-dropdown-noicon' : 'tabs');
+
+			outerNavClass = 'mw-portlet vector-menu';
+			if (navigation === 'mw-panel') {
+				outerNavClass += ' vector-menu-portal';
+			} else if (type === 'menu') {
+				outerNavClass += ' vector-menu-dropdown vector-dropdown vector-menu-dropdown-noicon';
```

Comment:
`vector-menu-dropdown-noicon` doesn't seem to have any effect anymore. I ended up removing it from MoreMenu. It's still being added server-side by [legacy Vector](https://gerrit.wikimedia.org/g/mediawiki/skins/Vector/+/8d6ef4769954dae3f084cc0281701a6818de5b29/includes/SkinVectorLegacy.php#113), and it's even being checked in acceptance tests, but it doesn't appear to be used anywhere else. I asked about it at [T291438#8503744](https://phabricator.wikimedia.org/T291438#8503744). For now I guess let's keep it.